### PR TITLE
Add support for system-level config (XDG_CONFIG_DIRS)

### PR DESCRIPTION
**Pull request by jirutka**

### CODE REVIEW

**Comment by artemsen** [comment]
Hi Jakub,
Thank you for the patch!
Could you add `Signed-off-by` to your commits?

File: src/config.c [hunk 1]
```diff
@@ -60,8 +60,8 @@ static FILE* open_file(void)
     size_t len;
     const char* postfix = "/swayimg/config";
     const size_t postfix_len = strlen(postfix);
+    char* config_dir = getenv("XDG_CONFIG_HOME");
```
Comment (artemsen) [inline src/config.c:63]: You don't need to remove `const`.

Comment (jirutka) [reply src/config.c]: Ah, right, `const` in C always confuses me.

**Comment by jirutka** [comment]
I fixed the `const` and added `Signed-off-by`.

**Pull request merged**